src/ekobox/__init__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 from ekomark import apply
 
-from . import evol_pdf, gen_info, gen_op, gen_theory
+from . import evol_pdf, info_file, operators_card, theory_card
```

Comment:
I guess `info_file` might even be just `info`, but also fine like this.

Reply:
let's keep like this